discord_bots/utils.py

```diff
@@ -238,6 +240,16 @@ async def update_next_map_to_map_after_next(rotation_id: str, is_verbose: bool):
     session.close()
 
 
+async def send_in_guild_message(guild, user_id, message_content=None, embed=None):
```

Comment:
nit: is it possible to add types to the parameters?

Reply:
yes of course